src/components/shared/ResultList.js

```diff
@@ -50,9 +51,14 @@ export default class ResultList extends Component<Props> {
         selected: index === selected
       })
     };
-
+    const svgName = item.tabExists
+      ? "source-icon tab"
+      : getSourceClassnames(item);
     return (
       <li {...props}>
+        <div className="svg">
```

Comment:
should we have this <div> if the svgName does not exist?

Reply:
I think `result-item-icon` may be a better className.

Reply:
Not 100% on the question from jason, but I'll take a stab at it
I would think the div should still be there to keep the spacing consistent?

```
| icon | someSource/source.js
sourceSource/typeWithoutSvg.weirdExtension  // compare below
|         | sourceSource/typeWithoutSvg.weirdExtension // compare above
| icon | someSource/source.js
```

@darkwing I'm fine with being more specific with the svgname. It's just not consistent with the other html elements defined in renderListItem (e.g. classnames for the other respective divs are just "title" and "subtitle")